examples/next-login/src/pages/home/login.tsx

```diff
@@ -0,0 +1,145 @@
+import { Text, Link, Stack } from '@ultraviolet/ui'
+import { Icon } from '@ultraviolet/icons'
+import {
+  Form,
+  TextInputField,
+  Submit,
+  CheckboxField,
+  useForm,
+  FormErrors,
+} from '@ultraviolet/form'
+import { Theme, css, useTheme } from '@emotion/react'
+import { useState } from 'react'
+
+type FormValues = {
+  email: string
+  password: string
+  remember: boolean
+}
+
+const emailRegex = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i
+const mockErrors: FormErrors = {
+  maxDate: ({ maxDate }) => `Date must be lower than ${maxDate?.toString()}`,
+  maxLength: ({ maxLength }) =>
+    `This field should have a length lower than ${maxLength}`,
+  minDate: ({ minDate }) => `Date must be greater than ${minDate?.toString()}`,
+  minLength: ({ minLength }) =>
+    `This field should have a length greater than ${minLength}`,
+  pattern: () => `This field should match the regex`,
+  required: () => 'This field is required',
+  max: ({ max }) => `This field is too high (maximum is : ${max})`,
+  min: ({ min }) => `This field is too low (minimum is: ${min})`,
+}
+
+const bodyStyle = (theme: Theme) => css`
+  .form-box {
+    margin: 5% 30% 5% 30%;
+    background: ${theme.colors.primary.background};
+    padding: 2%;
+  }
+
+  .inputs {
+    padding: 10px 30px 10px 30px;
+  }
+
+  .icon {
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .info-text {
+    margin-left: auto;
+    margin-right: auto;
+  }
+`
+const LogIn = () => {
+  const theme = useTheme()
+  const methods = useForm({
+    mode: 'onChange',
+    defaultValues: {
+      email: '',
+      password: '',
+      remember: false,
+    },
+  })
+
+  const [values, setValues] = useState(methods.watch())
+  const [loginText, setLoginText] = useState(<></>)
+  const handleSubmit = () => {
+    const val = methods.getValues()
+    setValues(val)
+    setLoginText(
+      <ul>
+        Welcome {val.email}. <li>Rember me checked : {String(val.remember)}</li>
+      </ul>,
+    )
+    setTimeout(() => setLoginText(<></>), 3000)
+    console.log('Values :', values)
+  }
+
+  return (
+    <div css={bodyStyle(theme)}>
+      <div className="form-box">
+        <Form<FormValues>
+          methods={methods}
+          errors={mockErrors}
+          onRawSubmit={handleSubmit}
+        >
+          <Stack gap={1}>
+            <div className="icon">
+              <Icon name="id" size="1.7em" />
+            </div>
+            <div>
+              <Text as="div" placement="center" variant="heading">
+                Login form
+              </Text>
+            </div>
```

Comment:
The div here has no purpose I believe 

```suggestion
              <Text as="div" placement="center" variant="heading">
                Login form
              </Text>
```